src/DB_Command.php

```diff
@@ -272,11 +284,17 @@ public function optimize( $_, $assoc_args ) {
 	 *     Success: Database repaired.
 	 */
 	public function repair( $_, $assoc_args ) {
-		self::run( Utils\esc_cmd( '/usr/bin/env mysqlcheck --no-defaults %s', DB_NAME ), array_merge( $assoc_args, array(
-			'repair' => true,
-		) ) );
+		self::run(
+			Utils\esc_cmd( '/usr/bin/env mysqlcheck --no-defaults %s', DB_NAME ),
+			array_merge(
```

Comment:
The `array_merge()` here is total overkill.
Can you rewrite this in the following way:
```php
$assoc_args['repair'] = true;
self::run(
	Utils\esc_cmd( '/usr/bin/env mysqlcheck --no-defaults %s', DB_NAME ),
	$assoc_args,
);
```